### DISCUSSION TITLE
Return type of createRouter is not RouterTyped

### DISCUSSION THREAD

**Discussion by ferferga** [posva/unplugin-vue-router#282] [Questions and Help]
When using createRouter as in the README example:

```
import { createRouter, createWebHistory } from 'vue-router/auto'

const router = createRouter({
  history: createWebHistory(),
})
```

The type of router is ``Router`` instead of ``RouterTyped``, which can cause type mismatchs

**Comment by posva** [comment]
I didn't mean to convert this into a discussion! I fixed it in a branch.
